clang/test/Analysis/Checkers/WebKit/unretained-members.mm

```diff
@@ -58,3 +59,11 @@ void forceTmplToInstantiate(FooTmpl<SomeObj, CFMutableArrayRef>) {}
 
   void forceTmplToInstantiate(RefPtr<SomeObj>) {}
 }
+
+@interface AnotherObject : NSObject {
+  NSString *ns_string;
+  // expected-warning@-1{{Instance variable 'ns_string' in 'AnotherObject' is a raw pointer to retainable type 'NSString'; member variables must be a RetainPtr}}
+  CFStringRef cf_string;
+  // expected-warning@-1{{Instance variable 'cf_string' in 'AnotherObject' is a retainable type 'CFStringRef'; member variables must be a RetainPtr}}
+}
+@end
```

Comment:
This is more of a question for my understanding: Do we expect the same behavior for raw pointers that are properties?
```
@interface AnotherObject : NSObject
@property(nonatomic,strong) NSString *prop_string;
@end
```

Reply:
I think so.

Reply:
Can we add a test case for properties?

Reply:
Turns out I have to enumerate properties separately from ivars. Added a test with code fix.